packages/integration-tests/index.js

```diff
@@ -663,18 +663,20 @@ async function getExecutions(workflowName, stackName, bucket, maxExecutionResult
  * @param {function} findExecutionFn - function that takes the taskInput and findExecutionFnParams
  * and returns a boolean indicating whether or not this is the correct instance of the workflow
  * @param {Object} findExecutionFnParams - params to be passed into findExecutionFn
+ * @param {integer} maxWaitSeconds - Set a custom wait time in seconds
  * @returns {undefined} - none
  */
 async function waitForTestExecutionStart(
   workflowName,
   stackName,
   bucket,
   findExecutionFn,
-  findExecutionFnParams
+  findExecutionFnParams,
+  maxWaitSeconds = maxWaitForStartedExecutionSecs
```

Comment:
I thought having more control over wait-time for a utility function like this might be useful. I'm open to opinions, though.

Reply:
I have another branch that turns these parameters into a parameter object